### PR TITLE
More tests for specific group

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
This PR adds test for creating a local mount for a specific group.

Related issue: #33679

## How Has This Been Tested?
Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 1]
```diff
@@ -678,20 +679,35 @@ public function theAdministratorCreatesTheLocalStorageMountForAUserUsingTheOccCo
 	}
 
 	/**
-	 * @When the administrator sets user :user as the applicable for last local storage mount using the occ command
+	 * @When /^the administrator (adds|removes) (user|group) "([^"]*)" (?:as|from) the applicable (?:user|group) for last local storage mount using the occ command$/
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: s/for last local storage mount/for the last local storage mount/

Let's put a `the` in here while we are waiting for drone to be sorted out.

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 2]
```diff
@@ -678,20 +679,35 @@ public function theAdministratorCreatesTheLocalStorageMountForAUserUsingTheOccCo
...
+	 * @Given /^the administrator has (added|removed) (user|group) "([^"]*)" (?:as|from) the applicable (?:user|group) for last local storage mount$/
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: s/for last local storage mount/for the last local storage mount/

File: tests/acceptance/features/cliMain/files.feature [hunk 3]
```diff
@@ -79,10 +79,90 @@ Feature: Files Operations command
...
+      | user0    |
+      | user1    |
+    And the administrator has created the local storage mount "local_storage2"
+    And the administrator has added user "user0" as the applicable user for last local storage mount
+    And the administrator has added user "user1" as the applicable user for last local storage mount
+    When the administrator removes user "user0" from the applicable user for last local storage mount using the occ command
+    Then as "user0" folder "/local_storage2" should not exist
+    And as "user1" folder "/local_storage2" should exist
+
+  Scenario: administrator should be able to create a local mount for a specific group
```
Comment (phil-davis) [inline tests/acceptance/features/cliMain/files.feature]: Is it possible to make multiple groups "applicable groups"?
If so, then there could be another scenario for that, with a user in each group.

And like for users, another scenario that makes 2 groups applicable groups, removes one of those groups, and checks that the expected users do and do not have access.

**Review by individual-it — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@paurakhsharma looks good. IMO the backport can be made now.

**Comment by phil-davis** [comment]
@PVince81 CI is green https://drone.owncloud.com/owncloud/core/13250 but the status has not come back to GitHub.
Please merge.

**Pull request merged**